src/Page.php

```diff
@@ -440,6 +441,48 @@ private function waitForReloadGenerator($eventName, $loaderId)
         }
     }
 
+    /**
+     * Wait until page contains Node.
+     *
+     * @param string $selectors
+     * @param int    $timeout
+     *
+     * @throws Exception\OperationTimedOut
+     */
+    public function waitUntilContainsElement(string $selectors, int $timeout = 30000) : self
```

Comment:
```suggestion
    public function waitUntilContainsElement(string $selectors, int $timeout = 30000): self
```
To make it pass on the styleci check.